### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Dominiquini** [sqlectron/sqlectron-gui#678]
The last version (1.37.0) is crashing when I try to open it!

Uncaught Exception:
Error: Cannot find module 'lodash.clonedeep'
Require stack:
- C:\Users\Rafael Dominiquini\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\db.js
- C:\Users\Rafael Dominiquini\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\index.js
- C:\Users\Rafael Dominiquini\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:261:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12789)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\Rafael Dominiquini\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\db.js:69:42)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)

**Windows 10 x64**

Thanks

**Comment by alejandro-00** [comment]
same problem:

A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module 'lodash.clonedeep'
Require stack:
- /opt/sqlectron/resources/app.asar/out/browser/core/db.js
- /opt/sqlectron/resources/app.asar/out/browser/core/index.js
- /opt/sqlectron/resources/app.asar/out/browser/main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:261:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12789)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/opt/sqlectron/resources/app.asar/out/browser/core/db.js:69:42)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)

Ubuntu 20.04.2LTS

**Comment by lvkins** [comment]
Same issue on Windows. Fresh `1.37.0` installation at Windows 10 x64:

```
[Window Title]
Error

[Main Instruction]
A JavaScript error occurred in the main process

[Content]
Uncaught Exception:
Error: Cannot find module 'lodash.clonedeep'
Require stack:
- C:\Users\username\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\db.js
- C:\Users\username\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\index.js
- C:\Users\username\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\main.js
- 
    at Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
    at Function.n._resolveFilename (electron/js2c/browser_init.js:261:1128)
    at Module._load (internal/modules/cjs/loader.js:732:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12789)
    at Module.require (internal/modules/cjs/loader.js:959:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\Users\username\AppData\Local\Programs\sqlectron\resources\app.asar\out\browser\core\db.js:69:42)
    at Module._compile (internal/modules/cjs/loader.js:1078:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1108:10)
    at Module.load (internal/modules/cjs/loader.js:935:32)

[OK]
```

**Comment by MasterOdin** [comment]
Fixed in 1f22bf678b7b278921d65ea9dfc159f7a200520e. Cutting a new release 1.37.1 momentarily.
